scripts/projects/TaikaiImporter.ts

```diff
@@ -0,0 +1,150 @@
+import { JSDOM } from 'jsdom';
+import { Project } from '../../src/types';
+import { GithubImporter } from './GithubImporter';
+import { Importer, detectParatimes, detectTags } from './Importer';
+
+export class TaikaiImporter implements Importer {
+  async importFromUrl(projectUrl: string): Promise<Project> {
+    console.log('Fetching URL:', projectUrl);
+    const projectUrlPath = projectUrl.split('/');
+    const projectId = projectUrlPath[projectUrlPath.length - 1];
+
+    const dom: JSDOM = await JSDOM.fromURL(projectUrl);
+    const document: Document = dom.window.document;
+    console.log('Document loaded, title:', document.title);
+
+    // Get project name from h1
+    const nameElement = document.querySelector('h1');
+    const name = nameElement?.textContent?.trim() || projectId;
+
+    // Get project description - extract content and convert HTML to markdown
+    const descriptionDiv = document.querySelector('div[class*="sc-"]') || document.querySelector('.xxmyC');
```

Comment:
The `.xxmyC` thing will break when they redeploy the site. Can you find something more permanent?